src/component/ItemDetails.js

```diff
@@ -1,13 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { FirestoreCollection, withFirestore } from 'react-firestore';
 import dayjs from 'dayjs';
-import { useParams } from 'react-router-dom';
+import { useParams, Redirect } from 'react-router-dom';
 import BackButton from './BackButton';
 
-const ItemDetails = ({ token, purchased }) => {
-  let { itemId } = useParams();
+const ItemDetails = ({ token, purchased, firestore }) => {
+  const [redirect, setRedirect] = useState(false);
+  const [isNewItem, setIsNewItem] = useState(true);
+  const { itemId } = useParams();
   const concatPath = `/lists/${token}/items`;
 
+  const confirmDeleteClick = event => {
+    if (window.confirm('Are you sure you wish to delete this item?')) {
+      deleteEntry(event.target.value);
+      setRedirect(true);
+    }
+  };
+
+  const deleteEntry = itemID => {
+    firestore
+      .collection('lists')
+      .doc(token)
+      .collection('items')
+      .doc(itemID)
+      .delete();
+  };
+
+  if (redirect) return <Redirect to="" />;
```

Comment:
There you go, no `else` after this `if` statement 👍

Reply:
I'm learning! 😁